.github/workflows/docs-ci.yaml

```diff
@@ -82,3 +85,6 @@ jobs:
 
       - run: make ${{ inputs.make-target }}
         working-directory: ${{ inputs.docs-directory }}
+
+      - run: make linkcheck
+        working-directory: ${{ inputs.docs-directory }}
```

Comment:
> I realized this workflow is used by more than `docs.nextstrain.org`, so before this can be merged, the following repos need to be audited for broken links:

An alternative is to make this opt-in. This can be achieved by specifying a new workflow input (e.g. `linkcheck-target` and conditioning this step on its presence:

```
- if: inputs.linkcheck-target
  run: make ${{ linkcheck-target }}
  working-directory: ${{ inputs.docs-directory }}
```

Usage would look like:

```
  docs-ci:
    uses: nextstrain/.github/.github/workflows/docs-ci.yaml@master
    with:
      environment-file: conda.yml
      docs-directory: .
      make-target: html
      linkcheck-target: linkcheck
```

Reply:
I've got momentum on this now, I'll just keep rolling through the other two repos...